app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/DefaultChannelAdapter.kt

```diff
@@ -65,13 +65,15 @@ class DefaultChannelAdapter(
             }
 
             itemView.setOnLongClickListener {
-                val builder = AlertDialog.Builder(context)
-                builder.setTitle(tile.url)
+                val builder = AlertDialog.Builder(context, R.style.DialogStyle)
+
+                val removeStr = context.resources.getString(R.string.homescreen_tile_remove)
+
+                val title = removeStr + " " + tile.title + "?"
```

Comment:
We'll need to get this localized again (I believe this could change the grammar is some languages), but we should wait until we get our final string from UX.  Would you mind adding a comment to that effect?  (e.g., 'we need to localize this, but are waiting until we hear back from UX'